### ISSUE TITLE
Make `EndpointPair` incomparable

### ISSUE THREAD

**Issue by jbduncan** [jbduncan/go-containers#3]
Blocked on https://github.com/jbduncan/go-containers/issues/3.

This can be done with `[0]func()` as per [this example](https://github.com/golang/go/blob/7abeefd2b1a03932891e581f1f90656ffebebce4/src/log/slog/value.go#L22-L22) in the golang/go repository.

**Comment by jbduncan** [comment]
Closed as this feature is no longer needed, due to `EndpointPair`s no longer being ordered and so no longer needing a custom `Equal` method.
